scripts/download_protos.sh

```diff
@@ -2,7 +2,6 @@
 
 declare -A URLS=(
   ["https://raw.githubusercontent.com/smswithoutborders/RelaySMS-Vault/main/protos/v1/vault.proto"]="protos/v1/vault.proto"
-  ["https://raw.githubusercontent.com/smswithoutborders/RelaySMS-Publisher/main/protos/v1/publisher.proto"]="protos/v1/publisher.proto"
```

Comment:
Why remove the publisher protobuffer file?

Reply:
Missed that 😄, thank you!